src/travis2docker/templates/build.sh

```diff
@@ -168,6 +168,108 @@ chown_all(){
     chown -R odoo:odoo /etc/postgresql-common/common-vauxoo.conf
 }
 
+configure_vim(){
+    if [ -z ${VIM_INSTALL+x} ];
+    then
+      return 0;
+    fi
+    # Upgrade & configure vim
+    apt install vim --only-upgrade
+    # Get vim version
+    VIM_VERSION=$(dpkg -s vim | grep Version | sed -n 's/.*\([0-9]\+\.[0-9]\+\)\..*/\1/p' | sed -r 's/\.//g')
+
+    wget -q -O /usr/share/vim/vim${VIM_VERSION}/spell/es.utf-8.spl http://ftp.vim.org/pub/vim/runtime/spell/es.utf-8.spl
+    git_clone_execute "${SPF13_REPO}" "3.0" "bootstrap.sh"
+    git_clone_copy "${VIM_OPENERP_REPO}" "master" "vim/" "${HOME}/.vim/bundle/vim-openerp"
+    git_clone_copy "${VIM_JEDI_REPO}" "master" "." "${HOME}/.vim/bundle/jedi-vim"
+
+    sed -i 's/ set mouse\=a/\"set mouse\=a/g' ~/.vimrc
+    sed -i "s/let g:neocomplete#enable_at_startup = 1/let g:neocomplete#enable_at_startup = 0/g" ~/.vimrc
+
+    # Disable virtualenv in Pymode
+    cat >> ~/.vimrc << EOF
```

Comment:
Use ${HOME} instead of ~ because it will be executed with root user but affecting the odoo folders

And remember to chown the pats created